src/Swagger/Serializer/ApiGatewayNormalizer.php

```diff
@@ -59,6 +59,8 @@ public function normalize($object, $format = null, array $context = [])
             $data['basePath'] = '/';
         }
 
+        unset($data['securityDefinitions']);
```

Comment:
Same question: why is this here?

Reply:
The way securityDefinitions is described is not compatible with AWS. `API Key security definition 'apiKey' has unexpected name or location. Ignoring.`

Reply:
@MLDMoritz Can you explain this change please? :smile:

Reply:
It's just a warning, so let's revert this change since it's not a blocker.

Reply:
AWS is not completely compliant to swagger spec, the securityDefinition must be extended to this to be imported without errors:

```"securityDefinitions": {
        "apiKey": {
            "type": "apiKey",
            "in": "header",
            "description": "Value for the Authorization header",
            "name": "Authorization",
            "x-amazon-apigateway-authtype": "awsSigv4"
        }
    },```

I could just hardcode that, but it won't do anything in the aws console because they implement custom authorizers, so I stripped it, because AWS doesn't make use of this at all.